### PR TITLE
Comments from Guy: May-12

### PR DESCRIPTION
**Pull request by ericvoit**

### CODE REVIEW

File: draft-voit-rats-attestation-results.md [hunk 1]
```diff
@@ -209,7 +217,8 @@ For more see {{claim-for-TEE-types}}.
 
 ### Attester
 
-Per {{I-D.ietf-rats-architecture}} Section 3.3, an Attester and a corresponding Attesting Environment might not share common boundaries.
+Per {{I-D.ietf-rats-architecture}} Figure 2, an Attester and a corresponding Attesting Environment might not share common boundaries.
+An example of such a case might be a laptop which records its security measurements within a TPM.
 In such cases, where connections are being established directly to an Attester but not to the Attesting Environment, the Verifier must include sufficient information in the Attestation Results to enable the Relying Party to have confidence that the Attester's trustworthiness is represented by Trustworthiness Claims signed by the appropriate Attesting Environment.
```
Comment (henkbirkholz) [inline draft-voit-rats-attestation-results.md]:
```suggestion
One example of such a case is a laptop that records its security measurements via the protected capabilities of a TPM.
In cases where connections are being established directly to an Attester but not to one of its Attesting Environments, a Verifier must include sufficient information in corresponding Attestation Results to enable Relying Parties to have confidence that the Attester's trustworthiness is represented via Trustworthiness Claims signed by the appropriate Attesting Environment.
```

File: draft-voit-rats-attestation-results.md [hunk 2]
```diff
@@ -220,11 +229,11 @@ Any of the above identities may be needed to be established by the Relying Party
 
 The mechanism for communicating the Attesting Environment identity (and if it is different, the Attester identity <!--Henk(old): there is a deterministic relationship here, see AuthSecID in interaction models -->) may be either implicit or explicit within an instance of Attestation Results.
 An example of explicit communication would be to include the following Identity Evidence directly in the Attestation Results: a unique identifier for an Attesting Environment, the name of a key which can be provably associated with that unique identifier, and the set of Attestation Results are signed using that key.
-An example of implicit communication would be to include the following Identity Evidence: a signature which has been made across the Attestation Results.
-It would be then up to the Relying Party's Appraisal Policy for Attestation Results to verify that this signature could only have come from an entity having access to the associated private key.
+An example of implicit communication would be to include the following Identity Evidence: a signature which has been made across the Verifier's Attestation Results.
```
Comment (henkbirkholz) [inline draft-voit-rats-attestation-results.md]:
```suggestion
An example of implicit communication would be to include the following Identity Evidence: a signature over all Attestation Results generated by a Verifier.
```

**Review by henkbirkholz — CHANGES_REQUESTED** [review]
some phrasing nits

**Comment by henkbirkholz** [comment]
A ref around 802.1AR seems to be broken at the moment.

File: draft-voit-rats-attestation-results.md [hunk 3]
```diff
@@ -196,10 +204,10 @@ For the Attesting Environment identity, there MUST exist a chain of trust ultima
 It is upon this root of trust that unique, non-repudiable identities may be founded.
 Example attested identities may include:
 
-* a type of hardware chip used for the Attesting Environment
-* a specific instance of a running Attesting Environment
-* a software build executing within an Attesting Environment
-* the developer(s) responsible for the code executing within an Attesting Environment
+* a type of hardware chip used for the Attesting Environment (e.g., TPM2.0)
+* a unique instance of a running Attesting Environment (e.g., LDevID {{802.1AR}}, Implementation ID {{-PSA}})
```
Comment (thomas-fossati) [inline draft-voit-rats-attestation-results.md]:
```suggestion
* a unique instance of a running Attesting Environment (e.g., LDevID {{802.1AR}}, Instance ID {{-PSA}})
```

Comment (ericvoit) [reply draft-voit-rats-attestation-results.md]: I initially placed Instance ID there.  But further reading led me to believe that multiple Instance ID's could be loaded into one TrustZone.  Could you confirm it is 1:1 between Attesting Environment and Instance ID?    I actually expect we are going to need to build more explicit mappings of identities in this section (or the Appendices) over time.

Comment (thomas-fossati) [reply draft-voit-rats-attestation-results.md]: Implementation ID is a kind of "class identifier" that is common to all devices sharing the same PSA RoT implementation.  Instance ID is the unique identifier of a PSA RoT instance.

File: draft-voit-rats-attestation-results.md [hunk 4]
```diff
@@ -387,11 +398,12 @@ Jump to step (6.1).
...
 As link layer protocols re-authenticate, steps (1) to (2) and steps (3) to (6) will independently refresh.
-In general, there are two types of triggers that trigger a refresh of Evidence generation (1), Attestation Result generation (2), and in consequence AR-augmented Evidence generation (4):
-1. live-cycle events, e.g. a change to an Authentication Secret of the Attester or an update of a software component
-2. uptime-cycle events, e.g. a hard reset of a composite device or a re-initialization of a TEE.
-3. authentication-cycle events, e.g. a link-layer interface resets or new TLS session is spawned.
-Appropriate re-generation of fresh Evidence, fresh Attestation Results, and fresh AR-augmented Evidence allows the trustworthiness of an Attester to be continuously re-appraised.
+This allows the Trustworthiness of Attester to be continuously re-appraised.
+There are only specific triggers which will refresh Evidence generation (1), Attestation Result generation (2), and in consequence AR-augmented Evidence generation (4):
+
+* live-cycle events, e.g. a change to an Authentication Secret of the Attester or an update of a software component
```
Comment (thomas-fossati) [inline draft-voit-rats-attestation-results.md:404]: non-native question: wouldn't this be life-cycle instead?

**Review by thomas-fossati — COMMENTED** [review]
LGTM!

File: draft-voit-rats-attestation-results.md (hunk 4 above)
Comment (ericvoit) [reply draft-voit-rats-attestation-results.md:404]: Yes

**Pull request merged**